app/Song-class.js

```diff
@@ -0,0 +1,29 @@
+function Song(data){
```

Comment:
Name the file exactly as the class it contains. fix stylistic issues: spacings of operators mainly